assets/src/utils/convert_service_utils.py

```diff
@@ -31,3 +31,27 @@ def post_to_convert(
         return False
     logger_.info(f"File {input_text} successfully converted")
     return True
+
+
+def post_pdf_to_convert(bucket: str, input_pdf, output_tokens) -> bool:
+    """
+    Puts pdf from html file into convert service
+    """
+    try:
+        response = requests.post(
+            url=settings.service_convert_pdf,
+            json={
+                "input_pdf": {"bucket": bucket, "path": input_pdf},
+                "output_tokens": {"bucket": bucket, "path": output_tokens},
+            },
+        )
+        if response.status_code != 201:
+            logger_.info(
+                f"File {input_pdf} failed to convert: " f"{response.text}"
```

Comment:
f-strings are not the best practice for logger.
See this: https://stackoverflow.com/questions/54367975/python-3-7-logging-f-strings-vs

Reply:
Please apply it to every logger usage